src/locales/ko/battle.ts

```diff
@@ -55,5 +55,8 @@ export const battle: SimpleTranslationEntries = {
   "notDisabled": "{{pokemonName}}의\n{{moveName}} 사슬묶기가 풀렸다!",
   "skipItemQuestion": "아이템을 받지 않고 넘어가시겠습니까?",
   "eggHatching": "어라…?",
-  "ivScannerUseQuestion": "{{pokemonName}}에게 개체값탐지기를 사용하시겠습니까?"
+  "ivScannerUseQuestion": "{{pokemonName}}에게 개체값탐지기를 사용하시겠습니까?",
+  "wildPokemonWithAffix": "Wild {{pokemonName}}",
+  "foePokemonWithAffix": "Foe {{pokemonName}}",
+  "useMove": "{{pokemonNameWithAffix}} used {{moveName}}!"
```

Comment:
```suggestion
  "wildPokemonWithAffix": "야생 {{pokemonName}}",
  "foePokemonWithAffix": "상대 {{pokemonName}}",
  "useMove": "{{pokemonNameWithAffix}}의 {{moveName}}!"
```
translated by the official translation(VIII, IX gen)